Formula/jigdo.rb

```diff
@@ -6,7 +6,8 @@ class Jigdo < Formula
   homepage "http://atterer.org/jigdo/"
   url "http://atterer.org/sites/atterer/files/2009-08/jigdo/jigdo-0.7.3.tar.bz2"
   sha256 "875c069abad67ce67d032a9479228acdb37c8162236c0e768369505f264827f0"
-  revision 6
+  license "GPL-2.0-only" => { with: "openvpn-openssl-exception" }
```

Comment:
From README:
```
Copyright (C) 2001-2006  |  richard@
Richard Atterer          |  atterer.net

  This program is free software; you can redistribute it and/or modify
  it under the terms of the GNU General Public License, version 2, as
  published by the Free Software Foundation.

  This program is distributed in the hope that it will be useful, but
  WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
  General Public License for more details.

  In addition, as a special exception, the author gives permission to
  link the jigdo code with the OpenSSL project's "OpenSSL" library (or
  with modified versions of it that use the same license as the
  "OpenSSL" library), and to distribute the linked executables. You
  must obey the GNU General Public License in all respects for all of
  the code used other than "OpenSSL". If you modify any jigdo source
  file, you may extend this exception to your version of the file, but
  you are not obliged to do so. If you do not wish to do so, delete
  this exception statement from your version.

Please note: The copyright notice in the file COPYING only applies to
the text of the GNU General Public License; the copyright of the
individual source files is as specified at the top of each file and
above. Also note that the code is licensed under GPL _version_2_ and
no other version. Special licensing for my (RA's) code is available on
request.
```